### PR TITLE
Norwegian translation of sensor_HassGetState.yaml

### PR DESCRIPTION
**Pull request by phellarv**
I have tried to translate sentences/en/sensor_HassGetState.yaml and tests/en/sensor_HassGetState.yaml to Norwegian. Constructive criticism is very much appreciated

### CODE REVIEW

**Comment by LaStrada** [comment]
Thanks for the PR! The PR is _huge_ so it will take some time to go through all changes. Also, go through the errors the linter complained about.

File: tests/nb/sensor_HassGetState.yaml [hunk 1]
```diff
@@ -0,0 +1,520 @@
+language: no
```
Comment (LaStrada) [inline tests/nb/sensor_HassGetState.yaml:1]:
```suggestion
language: nb
```

File: sentences/nb/sensor_HassGetState.yaml [hunk 2]
```diff
@@ -0,0 +1,619 @@
...
+          - "<what_is_the_class_of_name>"
+        response: one
+        requires_context:
+          domain: sensor
+          device_class: water
+        slots:
+          domain: sensor
+          device_class: water
+        expansion_rules:
+          class: "[Total] [mengde av] [forbrukt] vann|[totalt] vannforbruk)"
```
Comment (LaStrada) [inline sentences/nb/sensor_HassGetState.yaml:593]: I belive this one should be like this:
```suggestion
          class: "[Total] ([mengde av] [forbrukt] vann|[totalt] vannforbruk)"
```

**Review by LaStrada — CHANGES_REQUESTED** [review]
Still a few things more that needs to be done. E.g. missing expansion rule:
```
FAILED tests/test_language_sentences.py::test_sensor_HassGetState[nb] - hassil.recognize.MissingRuleError: Missing expansion rule <what_is_the_class_of_name>
```

**Comment by tetele** [comment]
@phellarv please mark your PR as ready for review if it's ready

**Comment by phellarv** [comment]
Sorry - Sick at the moment - Will work some more on it as soon as I get well.

**Comment by andebor** [comment]
@phellarv I took the liberty of drafting an initial version of the Norwegian translation of the `what_is_the_class_of_name` missing from your PR in https://github.com/home-assistant/intents/pull/1837. This PR also shows you how the fixtures relates to the test cases.

**Comment by phellarv** [comment]
> @phellarv I took the liberty of drafting an initial version of the Norwegian translation of the `what_is_the_class_of_name` missing from your PR in #1837. This PR also shows you how the fixtures relates to the test cases.

Excellent - I will pick up working on this as soon as my son leaves for University tuesday the coming week, and I will get back my PC. :-)

**Pull request closed without merging**